skyVegasTests/utilities/getValuesFromConsoleMessagesHelper.ts

```diff
@@ -0,0 +1,40 @@
+import { messageExists } from './messageExistsHelper';
+import {
+    deletePreviousConsoleMessages,
+    validateConsoleMessages,
+} from './validateConsoleMessagesHelper';
+
+export const currencyStringToNumber = (
+    currencyString: string | null
+): number => {
+    if (currencyString == null) {
```

Comment:
That will work here yes, we should make the change. One thing to keep in mind using if (!currencyString) is that it will trigger for 0 as well. In this case it is fine, because we are returning 0 anyway, but in other cases we may not want to use this and specifically check.